docs/navigation/results.md

```diff
@@ -97,31 +15,36 @@ from the `key` property of the request object.
 The target screen can then simply call `devliverNavigationResult(route.key, result)` on its own
```

Comment:
typo in `deliverNavigationResult`

Reply:
[e3d5a22](https://github.com/freeletics/khonshu/pull/827/commits/e3d5a221ef94283636d8b24c6e8fe2c32b004a52)